Source/Fuse.Controls.ScrollView/ScrollView.uno

```diff
@@ -377,7 +377,12 @@ namespace Fuse.Controls
 		*/
 		public void GotoRelative( float2 position )
 		{
-			Goto( (MaxScroll - MinScroll) * position + MinScroll );
+			Goto( RelativeToAbsolutePosition(position) );
+		}
+		
+		public float2 RelativeToAbsolutePosition( float2 pos )
```

Comment:
We refer to this as "absolute" elsewhere in the ScrollView, so I'd have to change all doc references. It's just a contrast to "relative" position"...

...but I'll just make this function internal so we don't have to worry about it now.